contrib/drivers/pgsql/pgsql_tables.go

```diff
@@ -32,6 +32,8 @@ WHERE
 ORDER BY
 	c.relname
 `
+
+	postgreRegex = regexp.MustCompile(`PostgreSQL (\d+\.\d+)`)
```

Comment:
It's better named as `versionRegex`, as it is used for version match.

Reply:
i'll change the name